source/chapters/djing_with_mixxx.rst

```diff
@@ -352,3 +352,130 @@ DJ as follows:
+The first and last sounds are determined by the first time the signal rises above -60 dBFS and the
+last time it goes below -60 dBFS.
+
+.. figure:: ../_static/intro-start-outro-end.png
+   :align: center
+   :alt: intro start marker placed at the first sound and outro end marker placed at the last sound
+   :figclass: pretty-figures
+
+   Intro start and outro end markers placed at the first and last sound of a track respectively by the analyzer
+
+You can decide where to place the intro end and the outro start points.
+Marking the whole intro and outro allows Mixxx to calculate how long each section is and
+show it on the overview waveform, this information can help you know when to press play on the next track.
+AutoDJ can also use this information to line up tracks and determine how long to crossfade.
+
+.. figure:: ../_static/intro-outro-full.png
+   :align: center
+   :alt: Intro and outro section on a track
+   :figclass: pretty-figures
+
+   Intro end and outro start markers placed on the track alongside the intro start and the outro end markers to make sections
+
+Mixing With Intro & Outro Cues
+------------------------------
+
+You can line up the ends of the intro and outro so the energy of the new song picks up
+right as the old track ends.
+
+.. figure:: ../_static/ends-aligned-waveforms.png
+   :align: center
+   :alt: Intro and outro ends aligned
+   :figclass: pretty-figures
+
+   Intro and outro end markers aligned showing on the waveform
+
+If the intro and outro are short enough to fit on the waveform at the same time, you can simply watch the
+waveforms and press play when the markers align. Otherwise, you can watch the time remaining on the deck, hover
+your mouse cursor over the outro end line on the overview waveform. Mixxx will show you how much time is left
+until that point. When that time equals the length of the intro, press play to precisely line up the end of the
+intro. If you have changed the tempo of the track, the time remaining scales to show the real time remaining at the
+adjusted tempo.
+
+Alternatively, you can line up the start of the intro & outro. In this case, watch the scrolling waveform when the
+outro start point is coming up on the old track. Then, press play on the new track when the old track reaches the outro start.
+
+   .. note:: There are no rules on which method should be used. The method anyone decides to use is an artistic judgement based
+             on the musical content of each track, what the vibe of the crowd is at that moment, and what you want to do with the mix.
+
+If you do not have the intro end and outro start points marked when loading a track, you can find and
+mark these in headphones before you mix in the track. Then seek back to the intro start to get the track ready. if you don't
+have time to find the outro end point before mixing in the track, you can use the new deck cloning feature to drag and drop to a new deck,
```

Comment:
This reminds
```suggestion
have time to find the outro end point before mixing in the track, you can use the deck cloning feature to drag and drop to a new deck,
```
If we call it "new", someone will have to remember to remove the "new" for the next Mixxx release, which will probably be forgotten.

This reminds me that we should add the deck cloning feature to the manual.